funnel/templates/email_project_registration.html.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "email_layout.html.jinja2" %}
+{% block content -%}
+
+<p>Hi {{ current_auth.user.fullname }},</p>
+
+<p>Your registration for <b>{{ project.title }}</b> was successful.</b></p>
+
+<p>The project is scheduled to begin at {{ project.schedule_start_at_localized|datetime }}</p>
```

Comment:
For a multi-day project, this could be in the past. Use the next upcoming session time instead. Also, sessions have a start time, projects don't. Your text should be "The next session is at..."